cmd/crc/cmd/start.go

```diff
@@ -87,10 +87,6 @@ func runStart(ctx context.Context) (*types.StartResult, error) {
 		EnableBundleQuayFallback: config.Get(crcConfig.EnableBundleQuayFallback).AsBool(),
 	}
 
-	if startConfig.Preset == preset.Podman {
-		logging.Warn(preset.PodmanDeprecatedWarning)
```

Comment:
Do we need to display a removal notice and refer people to use `podman machine` instead?
/cc: @crc-team

Reply:
how long will we show this message, is it going to be removed after a few releases? or we'll always show the message when user runs `crc config set podman preset`

another thing is this PR removes `preset.Podman` constant, i don't think we should keep this around to have this check and instead use `Preset("podman")`

Reply:
there are repeated non fatal errors complaining about the preset, both for `setup` and `start` and once we run `crc config unset preset` the errors go away

i think we need to inform the user how to get rid of the errors

```
% crc setup
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO unexpected preset mode podman, using default
ERRO unexpected preset mode podman, using default
INFO Using bundle path /Users/anath/.crc/cache/crc_vfkit_4.14.12_arm64.crcbundle
INFO Checking if running macOS version >= 13.x
INFO Checking if running as non-root
INFO Checking if crc-admin-helper executable is cached
INFO Checking if running on a supported CPU architecture
INFO Checking if crc executable symlink exists
INFO Checking minimum RAM requirements
INFO Check if Podman binary exists in: /Users/anath/.crc/bin/oc
INFO Checking if running emulated on Apple silicon
INFO Checking if vfkit is installed
INFO Checking if CRC bundle is extracted in '$HOME/.crc'
INFO Checking if /Users/anath/.crc/cache/crc_vfkit_4.14.12_arm64.crcbundle exists
INFO Checking if old launchd config for tray and/or daemon exists
INFO Checking if crc daemon plist file is present and loaded
INFO Checking SSH port availability
Your system is correctly setup for using CRC. Use 'crc start' to start the instance
ERRO unexpected preset mode podman, using default
```

---

```
% crc start --log-level debug
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
ERRO[0000] unexpected preset mode podman, using default
DEBU CRC version: 2.33.0+93a8fc
DEBU OpenShift version: 4.14.12
DEBU Podman version: 4.4.4
DEBU Running 'crc start'
ERRO unexpected preset mode podman, using default
DEBU Total memory of system is 34359738368 bytes
ERRO unexpected preset mode podman, using default
ERRO unexpected preset mode podman, using default
DEBU No new version available. The latest version is 2.33.0
ERRO unexpected preset mode podman, using default
DEBU Checking file: /Users/anath/.crc/machines/crc/.crc-exist
ERRO unexpected preset mode podman, using default
INFO Using bundle path /Users/anath/.crc/cache/crc_vfkit_4.14.12_arm64.crcbundle
INFO Checking if running macOS version >= 13.x
DEBU kern.osproductversion is: 14.3.1
INFO Checking if running as non-root
INFO Checking if crc-admin-helper executable is cached
DEBU Running '/Users/anath/.crc/bin/crc-admin-helper-darwin --version'
DEBU Found crc-admin-helper-darwin version 0.5.2
DEBU crc-admin-helper executable already cached
INFO Checking if running on a supported CPU architecture
DEBU GOARCH is arm64 GOOS is darwin
INFO Checking if crc executable symlink exists
INFO Checking minimum RAM requirements
DEBU Total memory of system is 34359738368 bytes
INFO Check if Podman binary exists in: /Users/anath/.crc/bin/oc
INFO Checking if running emulated on Apple silicon
INFO Checking if vfkit is installed
DEBU Running '/Users/anath/.crc/bin/vfkit -v'
DEBU Found vfkit version 0.5.0
INFO Checking if old launchd config for tray and/or daemon exists
INFO Checking if crc daemon plist file is present and loaded
DEBU Running 'bash -c launchctl list | grep com.redhat.crc.daemon | awk '{print $1}''
INFO Checking SSH port availability
DEBU Checking file: /Users/anath/.crc/machines/crc/.crc-exist
INFO Loading bundle: crc_vfkit_4.14.12_arm64...
ERRO unexpected preset mode podman, using default
DEBU Using secret from configuration
INFO Creating CRC VM for OpenShift 4.14.12...
```

Reply:
would be useful to have a removal notice for a while. We can remove it when we remember about it :)

Reply:
I would suggest disallowing starting with this. A removal and alternative use message in that case, right?

Reply:
I think we can return an error when user tries to start an existing machine with podman preset and modify tests to expect that error